projects/web-wallet/rpc-protocol.md

```diff
@@ -0,0 +1,330 @@
+# 1. Introduction
+
+This document outlines the remote procedure call (RPC) methods for a decentralized application (dApp) to interact with Hathor wallets.
+
+The design is currently a minimum viable product (MVP), addressing only known needs from our community. It should be noted that this API design may change as the platform evolves and additional needs arise.
+
+The idea here is to partially replicate the headless wallet API as it is already being used by many partners and has been validated to work on their use cases
+
+# 2. Reference-level explanation
+
+## 2.1 RPC Methods
+
```

Comment:
What about a method to get an address? For instance, the website may want to get the user's address to send some coins.

This would be equivalent to the user copying the address from the wallet and pasting on the website.

Reply:
I like the idea, added the RPC method here: 8fd28e7a9bfed60cb7f24b247b03bc057e746b09

The wallet should decide whether to let the user pick an address or return an arbitrary one